source/Canvas.js

```diff
@@ -17,6 +17,7 @@ class Canvas {
 
   computeCartogram(options) {
     this._mapGraphic.computeCartogram(options)
+    this.getCartogramArea()
```

Comment:
I don't think this call does anything.

BUT I was also going to say that, instead of recomputing the area every time the slider changes, it would make sense to compute it once here (in `MapGraphic`) and memoize it.